src/customizations/volto/components/manage/Blocks/Image/Edit.jsx

```diff
@@ -196,213 +147,206 @@ class Edit extends Component {
             filename: file[0].name,
           },
         },
-        this.props.block,
+        props.block,
       );
     });
   };
 
-  /**
-   * Keydown handler on Variant Menu Form
-   * This is required since the ENTER key is already mapped to a onKeyDown
-   * event and needs to be overriden with a child onKeyDown.
-   * @method onKeyDownVariantMenuForm
-   * @param {Object} e Event object
-   * @returns {undefined}
-   */
-  onKeyDownVariantMenuForm = (e) => {
+  // /**
```

Comment:
@andreiggr the code was already commented fine there is no reason to have inline comments